src/icons/Status.js

```diff
@@ -12,27 +14,15 @@ export function Valid({ className }) {
     return (
         <svg
             xmlns="http://www.w3.org/2000/svg"
-            width="48"
-            height="48"
             viewBox="0 0 48 48"
             className={className}
         >
             <path d="M24 4C12.95 4 4 12.95 4 24c0 11.04 8.95 20 20 20 11.04 0 20-8.96 20-20 0-11.05-8.96-20-20-20zm-4 30L10 24l2.83-2.83L20 28.34l15.17-15.17L38 16 20 34z" />
-            <style jsx>{`
-                svg {
-                    fill: ${theme.valid};
-                    width: 24px;
-                    height: 24px;
-                    margin-right: ${spacers.dp4};
```

Comment:
Seems I didn't just remove the fill style but also the `margin-right: ${spacers.dp4};`. I don't actually think that this style belongs here. If I remember it correctly, we want to end up with an icon repo too. Components that use these icons should expect them to all behave the same, and it should be possible for these components to control things like fill, margin and size.